src/translations/screens/subscreens/PurchaseOverview.ts

```diff
@@ -210,7 +210,7 @@ const PurchaseOverviewTexts = {
     ),
     button: {
       payment: _('Til betaling', 'To payment', 'Til betaling'),
-      sendToOthers: _('Gå videre', 'Continue', 'Gå vidare'),
+      sendToOthers: _('Select recipient', 'Select recipient', 'Vel mottakar'),
```

Comment:
The first `'Select recipient', 'Select recipient'` should be in Norwegian.

Reply:
![image](https://github.com/user-attachments/assets/a504f253-6b76-483d-b775-8baf7b1890c2)